File: tuts/012-transitgateway-gettingstarted/kotlin/src/main/kotlin/com/example/ec2/GettingStartedScenario.kt
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package com.example.ec2

import aws.sdk.kotlin.services.ec2.Ec2Client
import kotlinx.coroutines.runBlocking

fun main() = runBlocking {
    Ec2Client { region = "us-east-1" }.use { client ->
        val wrapper = Ec2Wrapper(client)
        println("Running Ec2 getting started scenario...")
        // TODO: setup, interact, teardown
    }
}

File: tuts/002-vpc-gs/swift/Sources/GettingStartedScenario.swift
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import AWSEc2

@main
struct GettingStartedScenario {
    static func main() async throws {
        let client = try Ec2Client(region: "us-east-1")
        let wrapper = Ec2Wrapper(client: client)
        print("Running Ec2 getting started scenario...")
        // TODO: setup, interact, teardown
        let _ = wrapper
    }
}
